react-ui/web/src/components/monitoring/dashboard/Container.js

```diff
@@ -150,13 +153,16 @@ function Container (props) {
                     </Grid>
                     
                     <Grid item md={9}>
+                        <ShiftsPanel ShiftData={monitoringShifts}/>
                         <TabBar
                             chosenTab={chosenTab}
                             onSelect={handleTabSelected}
                             tabsArray={tabs_array}
                         />
+                        
 
                         <div className={classes.tabBarContent}>
+                            
```

Comment:
```suggestion

                        <div className={classes.tabBarContent}>
```

remove lang yung extra spaces